src/main/webapp/app/admin/admin-container/admin-container.component.scss

```
@@ -2,68 +2,49 @@ $menu-width-closed: 64px;
     width: 100vw;
     padding: 0 $spacing 0 0;
 }
 
-.mat-drawer-side {
-    border: none;
-}
-
-.mat-drawer-container {
+.layout-container {
+    display: flex;
     // Fixed height based on viewport minus navbar and footer
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)